pyanaconda/startup_utils.py

```diff
@@ -183,8 +183,23 @@ def setup_logging_from_options(options):
             log.error("Could not setup remotelog with %s", options.remotelog)
 
 
-def prompt_for_ssh():
-    """Prompt the user to ssh to the installation environment on the s390."""
+def prompt_for_ssh(options):
+    """Prompt the user to ssh to the installation environment on the s390.
+
+    :param options: Anaconda command line/boot options
+    :return: True if the prompt is printed, otherwise False
+    """
+    if not blivet.arch.is_s390():
+        return False
```

Comment:
If it's s390x-only, why not have that in the function name?

Reply:
It doesn't have to be s390x only in the future.